### ISSUE TITLE
Is it possible to drop QT-dependency?

### ISSUE THREAD

**Issue by GitMensch** [maphew/svg-explorer-extension#44]
This issue is primarily intended as a real question - is there anything else QT is used for than the actual SVG->"preview" part? Can this be done with a more lightweight dependency, too?

I *guess* the "os-portable" part of QT is completely irrelevant for this "Windows Explorer" plugin, isn't it?

**Comment by tibold** [comment]
Yes. The only reason Qt is part of this is because when the library was originally written I was working on a desktop app written in Qr 4.x and I had difficulties getting librsvg running. :) With Qt I had up to date experience to render SVGs. Also the previews I needed to see how the SVGs would be rendered in a Qt based app since I was approving SVG icons for a Qt app.

If we can find working samples on either librsvg/cairo or any other SVG rendering framework that works better than Qt, I'd be more than happy to replace it.

**Comment by jonshipman** [comment]
@GitMensch I know that there isn't currently a build of Qt for Windows on ARM. So that problem would be solved by dropping Qt.
